examples/null/resource.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: null.template.jet.crossplane.io/v1alpha1
+apiVersion: null.tencentcloud.jet.crossplane.io/v1alpha1
```

Comment:
Did we need null resource in example folder ? Think we can remove it

Reply:
Did we need null resource ? Think you can remove

Reply:
I check null had been deleted.
<img width="1762" alt="image" src="https://user-images.githubusercontent.com/26042144/196829129-6c574eee-977e-486b-a497-ec61d8e6f693.png">